airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/base/IntegrationRunner.kt

```diff
@@ -110,17 +109,21 @@ internal constructor(
 
     @Trace(operationName = "RUN_OPERATION")
     @Throws(Exception::class)
-    fun run(args: Array<String>) {
+    @JvmOverloads
+    fun run(args: Array<String>, exceptionHandler: ConnectorExceptionHandler? = null) {
         val parsed = cliParser.parse(args)
         try {
-            runInternal(parsed)
+            runInternal(parsed, exceptionHandler)
         } catch (e: Exception) {
             throw e
         }
     }
 
     @Throws(Exception::class)
-    private fun runInternal(parsed: IntegrationConfig) {
+    private fun runInternal(
+        parsed: IntegrationConfig,
+        exceptionHandler: ConnectorExceptionHandler? = null
```

Comment:
make the default exception handler the default value here?

Reply:
yes!

Reply:
fixed.